install/installing-custom-components.md

```diff
@@ -0,0 +1,198 @@
+[6]: https://github.com/knative/build/releases/tag/v0.2.0
+[6.1]: https://github.com/knative/eventing/releases/tag/v0.2.0
+[7]: https://github.com/knative/eventing/releases/download/v0.2.0/eventing.yaml
+[8]: https://github.com/knative/eventing-sources/releases/download/v0.2.0/release.yaml
+[8.1]: https://github.com/knative/eventing-sources/releases/tag/v0.2.0
+[9]: https://github.com/knative/serving/releases/download/v0.2.1/serving.yaml
+
+Note: Each of the installable Knative resources are also listed in the Assets
+section of each Knative component's release page:
+* [Serving v0.2.1](https://github.com/knative/serving/releases/tag/v0.2.1)
+* [Build v0.2.0](https://github.com/knative/build/releases/tag/v0.2.0)
+* [Eventing v0.2.0](https://github.com/knative/eventing/releases/tag/v0.2.0)
+
+### Installing individual Knative components
+
+**Tip**: From the table above, you can copy and paste the URL and filename of
+the installation package that you want to install.
+
+1. To install a specific component, run the `kubectl apply` command and specify the
+   installation package filename:
+
+    ```bash
+    kubectl apply --filename https://github.com/knative/[COMPONENT]/releases/download/[VERSION]/[FILENAME].yaml
+    ```
+
+    where `[COMPONENT]`, `[VERSION]`, and `[FILENAME]` are the Knative component, release version, and filename of the installable resource. Examples:
+    * `https://github.com/knative/build/releases/download/v0.2.0/release.yaml`
+    * `https://github.com/knative/eventing/releases/download/v0.2.0/eventing.yaml`
+    * `https://github.com/knative/eventing-sources/releases/download/v0.2.0/release.yaml`
+    * `https://github.com/knative/serving/releases/download/v0.2.1/serving.yaml`
+
+1. Depending on what you choose to install, you view the status of your
+   installation by running one or more of the following commands. It
+   might take a few seconds so rerun the commands until all of the components
+   show a `STATUS` of `Running`:
+
+    ```bash
+    kubectl get pods --namespace knative-build
+    kubectl get pods --namespace knative-eventing
+    kubectl get pods --namespace knative-eventing-sources
+    kubectl get pods --namespace knative-serving
+    ```
+
+    > Tip: You can append the `--watch` flag to the `kubectl get` commands to
+      view the pod status in realtime. You use `CTRL + C` to exit watch mode.
+
+1. If you installed one of the installation packages that include an observability
+   plugin, you can run the following command to ensure that the `knative-monitoriing`
+   pods show a `STATUS` of `Running`:
+
+    ```bash
+    kubectl get pods --namespace knative-monitoring
+    ```
+
+You are now ready to deploy an app, create a build, or start sending and receiving
+events in your Knative cluster.
+
+## What's next
+
+Depending on which of the Knative components that you chose to install, you can
+use one of the following documents to help get you started in Knative:
```

Comment:
"Depending on which of the Knative components that you chose to install, you can
use one of the following documents to help get you started in Knative:" --> "Depending on the Knative components you installed, the following guides can help you get started with Knative:"

Reply:
done